Cargo.toml

```diff
@@ -43,3 +43,6 @@ chrono = "0.4.39"
 ratatui = "0.27.0"
 crossterm = "0.28.1"
 serde_json = "1.0.122"
+
+[profile.release]
```

Comment:
Thanks for taking a stab at this!

Would you mind explicitly enabling it for all default profiles (dev, release, test, and bench) here instead of just release?

Reply:
For sure, just updated my commit